spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -56,8 +58,16 @@ trait CommonJobExecution extends ProjectMetadata {
 
   protected val log: Logger = LoggerFactory.getLogger(this.getClass)
   protected val configReader: ConfigReader = new ConfigReader()
+
   protected val menasBaseUrls: List[String] = MenasConnectionStringParser.parse(configReader.getString("menas.rest.uri"))
   protected val menasUrlsRetryCount: Option[Int] = configReader.getIntOption("menas.rest.retryCount")
+  protected val optionallyRetryableExceptions: Set[OptRetryableExceptions] =
+    configReader
+      .getIntListOption("menas.rest.optionallyRetryableExceptions")
+      .getOrElse(Set.empty)
+      .toSet
+      .map(mapIntToOptionallyRetryableException(_: Int))
```

Comment:
I know, I just tried to be explicit. But since Scala compiler is pretty powerful, and should be able to check this during compile time, I can remove this.